app/components/publications/publications-table-row.hbs

```diff
@@ -158,48 +210,52 @@ request the whole row and not only the arrow at the end to be clickable }}
       {{/if}}
     </td>
   {{/if}}
-  {{#if @tableColumnDisplayOptions.publicationReceivedDate}}
-    <td class="auk-u-text-nowrap">
-      {{#if @publicationFlow.publicationSubcase.receivedDate}}
-        {{moment-format
-          @publicationFlow.publicationSubcase.receivedDate
-          "DD-MM-YYYY"
-        }}
-      {{else}}
-        {{t "dash"}}
-      {{/if}}
-    </td>
-  {{/if}}
-  {{#if @tableColumnDisplayOptions.translationReceivedDate}}
-    <td class="auk-u-text-nowrap">
-      {{#if @publicationFlow.translationSubcase.receivedDate}}
-        {{moment-format
-          @publicationFlow.translationSubcase.receivedDate
-          "DD-MM-YYYY"
-        }}
+  {{#if @tableColumnDisplayOptions.publicationDueDate}}
+    <td
+      data-test-publication-table-row-publication-due-date
+      class="auk-u-text-nowrap"
+    >
+      {{#if @publicationFlow.publicationSubcase.dueDate}}
+        {{!Ember seems not to await async getter}}
```

Comment:
Dat klopt. Op sommige plaatsen in de code wordt hiervoor gebruik gemaakt van een [PromiseObject](https://api.emberjs.com/ember-data/3.26.0/classes/PromiseObject), waardoor de view wel update. Maar we trachten dit patroon te vermijden. Het verleden heeft geleerd dat dat leidt tot een oncontroleerbare stroom van network requests, zeker wanneer getters op elkaar gaan dependen.